app/src/main/res/layout/filter_dialog.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingLeft="20dp"
+    android:paddingTop="25dp"
+    android:paddingRight="20dp"
+    android:paddingBottom="0dp">
```

Comment:
Material Design uses multiples of `8dp` for spacing (see https://material.io/design/layout/spacing-methods.html#baseline-grid). Other dialogs in AntennaPod use `16dp` for padding.

Reply:
ok?

Reply:
:+1: